src/test/java/tutorease/address/logic/commands/DeleteContactCommandTest.java

```diff
@@ -32,6 +34,41 @@ public void constructor_nullIndex_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new DeleteContactCommand(null));
     }
 
+    @Test
+    public void testToString_returnsCorrectString() {
+        Index index = Index.fromZeroBased(0);
+        DeleteContactCommand deleteCommand = new DeleteContactCommand(index);
+
+        // Test toString method
+        assertEquals("DeleteContactCommand{targetIndex=tutorease.address.commons.core."
+                + "index.Index{zeroBasedIndex=0}}", deleteCommand.toString());
+    }
+
+    @Test
+    public void equals() {
+        Index index = Index.fromZeroBased(0);
+        DeleteContactCommand deleteCommand = new DeleteContactCommand(index);
+        // Test the same object
+        assertEquals(deleteCommand, deleteCommand);
+
+        Index index1 = Index.fromZeroBased(1);
+        DeleteContactCommand deleteCommand1 = new DeleteContactCommand(index1);
+        DeleteContactCommand deleteCommand2 = new DeleteContactCommand(index1);
+
+        // Test different objects but same Index
+        assertEquals(deleteCommand1, deleteCommand2);
+
+        Index index2 = Index.fromZeroBased(2);
+        DeleteContactCommand deleteCommand11 = new DeleteContactCommand(index1);
```

Comment:
Could perhaps consider changing the naming of the variables here to something more intuitive and readable. For eg. deleteCommandWithIndex1 and deleteCommandWithIndex2

Reply:
Agreed. Other than that LGTM!

Reply:
Good read on this. But otherwise LGTM 👍

Reply:
Fixed:)